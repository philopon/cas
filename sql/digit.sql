SELECT cas_first('42221-12-5'::cas);
SELECT cas_second('42221-12-5'::cas);
SELECT cas_checkdigit('42221-12-5'::cas);
