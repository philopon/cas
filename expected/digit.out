SELECT cas_first('42221-12-5'::cas);
 cas_first 
-----------
     42221
(1 row)

SELECT cas_second('42221-12-5'::cas);
 cas_second 
------------
         12
(1 row)

SELECT cas_checkdigit('42221-12-5'::cas);
 cas_checkdigit 
----------------
              5
(1 row)

